app.rb

```diff
@@ -24,6 +25,14 @@ def select_option(user_response)
       create_rental
     when 6
       rental_by_id
+    when 7
+      books_json = @book_options.books_objects.to_json
+      person_json = @person_options.people_objects.to_json
+      rentals_json = @rental_options.rentals_objects.to_json
+
+      File.open('db/books.json', 'w') { |file| file.write(books_json) }
+      File.open('db/people.json', 'w') { |file| file.write(person_json) }
+      File.open('db/rentals.json', 'w') { |file| file.write(rentals_json) }
     else
       puts 'Please add a valid number'
```

Comment:
@Metaverse-Nabeel That case is too long. We need to separate all that logic in a different method. Something like store_json_data. Then we can execute the method inside the case 7

Reply:
Done